dev-manuals/test-plans/lecturer-interface-frontend/plans/u-lecturer-interface-12-change-game-of-minigame.md

```diff
@@ -0,0 +1,36 @@
+# Lecturer-Interface: Change game of minigame (`u.lecturer-interface-12`)
+
+
+Version: V1.0, 09.08.2022 \
+Author: Max Kästner
+
+## Description
+
+The game of a minigame can be changed in a dropdown. For minigames of a world the path is `/courses/<course-id>/worlds/<world-index>/minigames` and for a dungeon `/courses/<course-id>/worlds/<world-index>/dungeons/<dungeon-index>/minigames`.
+
+## Precondition
+
+At least one course is created. The tester is on the view of an area minigames list as described in the description above.
+
+## Postcondition
+
+A game of a minigame was changed.
+
+## Typical procedure
+
+1. Click on a dropwdown of a minigame to change the game
+2. Select a specific game
+3. A feedback toast message appears which contains that game was updated
+4. The game is now displayed in the dropdown
+
+## Alternative procedures
+
```

Comment:
```suggestion

2.1. Cancel the edit by closing the dropdown without selecting a game \
3.1. Select the already selected game - nothing happens \
3.2. Select the already selected game - a warning is displayed
```

Reply:
one same thing but two different reactions?

Reply:
I've taken the freedom of interpreting `alternative procedures` as `how could we plausibly react in this error/ edge case?` instead of `how do we react to this error/ edge case at the moment?`.
I don't know if everything in the alternative procedures must be implemented, I would have said it is also possible to list other plausible, but not chosen, error handlings.

Reply:
```suggestion
2.1. Cancel the edit by closing the dropdown without selecting a game \
3.1. Select the already selected game - nothing happens
```